libs/ui/src/components/MediaItem/MediaItem.vue

```diff
@@ -26,8 +26,8 @@
     <div
       v-if="isInteractive"
       class="k-shade border-k-grey is-flex is-align-items-center is-justify-content-center border is-rounded absolute-position image is-24x24">
-      <div class="is-size-7 is-text-weight-500">
-        {{ '<' + '/' + '>' }}
+      <div class="is-size-7 has-text-weight-medium">
+        {{ interactiveIconTitle }}
```

Comment:
or use icon [code](https://fontawesome.com/icons/code?f=classic&s=solid)?

Reply:
Thank you. It also works.